packages/quantic/cypress/jsconfig.json

```diff
@@ -1,3 +1,4 @@
 {
-  "include": ["../../node_modules/cypress", "./**/*.js"]
+  "include": ["../../node_modules/cypress", "./**/*.js"],
+  "typeAcquisition": {"include": ["cypress"]}
```

Comment:
So this fixing the Cypress typing errors?

very nice 👌

Reply:
Yes! I finally found a solution! This and the change to the tsconfig

I think this one is the major fix `"include": ["../node_modules/cypress"` in the tsconfig that's under the cypress folder.

I believe this means the cypress code is included in the steps of compiling the TS for the cypress tests, so it replaces the default jest types that are brought from the other files? 🤷 

It comes from the Cypress doc and their example repo 
https://docs.cypress.io/guides/tooling/typescript-support#Clashing-types-with-Jest
https://github.com/cypress-io/cypress-and-jest-typescript-example